components/SuccessfulTxDialog.qml

```diff
@@ -142,14 +142,35 @@ Rectangle {
             fontSize: 16
         }
 
-        // open folder / done buttons
+        // view progress / open folder / done buttons
         RowLayout {
             id: buttons
             spacing: 70
             Layout.alignment: Qt.AlignBottom | Qt.AlignHCenter
             Layout.fillWidth: true
             Layout.preferredHeight: 50
 
+            MoneroComponents.StandardButton {
+                id: viewProgressButton
+                visible: !appWindow.viewOnly
+                text: qsTr("View progress") + translationManager.emptyString;
+                width: 200
+                primary: false
+                KeyNavigation.tab: doneButton
+                Keys.enabled: viewProgressButton.visible
+                Keys.onReturnPressed: viewProgressButton.onClicked
+                Keys.onEnterPressed: viewProgressButton.onClicked
```

Comment:
Not needed. `StandardButton` should handle the keys internally. If it doesn't work, just fix the `StandardButton`.